### PR TITLE
Add wrapper to help logging episode returns

### PR DESCRIPTION
**Pull request by ffelten**
The implementation is mostly a copy paste of the original gym. I had to copy paste instead of override and call to super because the way the return is a numpy array, which is mutable, and the original implementation resets it to 0. Hence, if we kept the original, the return will always be a vector of zeros (because resetted)

### CODE REVIEW

File: mo_gym/utils.py [hunk 1]
```diff
@@ -111,3 +115,64 @@ def __init__(
...
+
+        Args:
+            env (Env): The environment to apply the wrapper
+            deque_size: The size of the buffers :attr:`return_queue` and :attr:`length_queue`
+        """
+        super().__init__(env, deque_size)
+        # Here we just override the standard implementation to extend to MO
+        self.reward_dim = self.env.reward_space.shape[0]
+
+    def reset(self, **kwargs):
```
[the remark this answers is not in the document]
Comment (ffelten) [reply mo_gym/utils.py:131]: That's what Gym does for all wrappers: rely on kwargs to forward pass it.

File: mo_gym/utils.py [hunk 2]
```diff
@@ -111,3 +115,64 @@ def __init__(
...
+        return observations
+
+    def step(self, action):
+        """Steps through the environment, recording the episode statistics."""
+        # This is the code from the RecordEpisodeStatistics wrapper from gym.
+        observations, rewards, dones, infos = self.env.step(action)
+        assert isinstance(
+            infos, dict
+        ), f"`info` dtype is {type(infos)} while supported dtype is `dict`. This may be due to usage of other wrappers in the wrong order."
+        self.episode_returns += rewards
```
[the remark this answers is not in the document]
Comment (ffelten) [reply mo_gym/utils.py:144]: Yeah, it is weird Gym is not doing it. I wonder why.

File: mo_gym/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (LucasAlegre) [reply mo_gym/utils.py:132]: But then the variable observations might be a tuple, right?

File: mo_gym/utils.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (LucasAlegre) [reply mo_gym/utils.py:146]: True, people tend to only see the total reward, when we are actually optimizing discounted return.

**Pull request merged**